### ISSUE TITLE
Pre-processed data 

### ISSUE THREAD

**Issue by wishinghao** [xz-pisces/DGarment#1]
Hello, the data provided in the cloud storage (netB) seems to be a network rather than preprocessed data.

**Comment by xz-pisces** [comment]
yes. netB is the weight file of pifuhd, use /person/pifuhd.py to estimate the normals
